Makefile

```diff
@@ -45,6 +45,10 @@ install:
 	@echo "  >  \033[32mInstalling bridge...\033[0m "
 	cd cmd/chainbridge && go install
 
+install-mkdocs:
+	@echo "  >  \033[32mGetting mkdocs image ...\033[0m "
+	docker pull squidfunk/mkdocs-material
+
```

Comment:
You don't really need this, `docker run` should pull the image for you. It might be good to include the command to generate the html files (`mkdocs build --site-dir <dir>`) and to install mkdocs (pip instal...)